README.md

```diff
@@ -21,15 +21,27 @@ We're working on [tutorials](https://napari.org/tutorials/), but you can also qu
 
 ## installation
 
+### which distribution to install
+If you want to contribute back to napari codebase, you need to install from source code, see [from source](#from-source)
+
+If you do not want to use napari as python code and only use it as GUI app, the bundled app is the easiest way to install, 
+this is also the only method that does not require python knowledge to work with napari, see [from bundled app](https://napari.org/tutorials/fundamentals/installation#installing-as-a-bundled-app)
+
+If you are using napari while using python to programmatically interact with the app, you can either install via pip or source code, 
+however we recommend that you use conda to help managing the virtual environment, otherwise you may see compilation issues that are specific to machines.
+
+
 ### from pip, with "batteries included"
 
 napari can be installed on most macOS, Linux, and Windows systems with
-Python 3.7 and 3.8 using pip:
+Python 3.7 and 3.8 using pip. However, for windows users, you need to preinstall [Microsoft Visual C++ Build Tools](https://visualstudio.microsoft.com/downloads/)
+in order to install vispy (one of the dependency packages) on windows machines.
```

Comment:
```suggestion
Python 3.7 and 3.8 using pip. However, for Windows users, you need to preinstall [Microsoft Visual C++ Build Tools](https://visualstudio.microsoft.com/downloads/)
in order to install VisPy (one of the packages we depend on) on Windows machines.
```

Also, I *think* the above is only true for Windows 32-bit Python. For 64-bit, the wheels provided by VisPy and dependencies *should* be sufficient? Maybe @djhoese can weigh in here. This seems related to a [recent discussion](https://gitter.im/vispy/vispy?at=5f90c884a7e77a0ff160acc1) we had on the VisPy gitter.

Reply:
I don't think this is true anymore but please correct me. If this is addressing the semi-recent issue where vispy didn't have a Python 3.8 wheel, then this should now be resolved with the vispy 0.6.5 release (and wheels that came as part of that distribution).

If this is about freetype-py, one of vispy's dependencies then yes this is only for 32-bit Windows installations and this is not a VisPy wheel thing, this is a freetype-py wheel thing. You install vispy (however you want to) and it tells pip to also install freetype-py. On 32-bit Windows systems pip will not find a wheel for freetype-py so it will need to install from source (freetype-py's sdist). By default, the freetype-py's sdist installation does not include a freetype C library binary so it tries to compile against a system installed one which will likely fail. It can fail because either freetype C is not installed or the user doesn't have a compiler. Actually...freetype-py might be pure python (ctypes) and not need a compiler.

https://github.com/rougier/freetype-py

Reply:
See these installation instructions: https://github.com/rougier/freetype-py#installation-with-compiling-freetype-from-source

Looking at the setup.py it looks like you just need to define that environment variable and it will **build** the freetype C library. However, this probably means you need a compiler.

Reply:
I am not sure when we are going to upgrade vispy, so for now I will include this to be safe, we can definitely take it out once we are ready to upgrade vispy version